assets/javascripts/lib/page.coffee

```diff
@@ -210,7 +212,7 @@ track = ->
     # Only ask for consent once per browser session
     Cookies.set('analyticsConsentAsked', '1')
```

Comment:
How does your change relate to those two lines? You to tackle the same problem.

Reply:
This line avoids show me more than once the analitics consent when cookies are enabled, In my case I do not accept neither the devdocs cookie nor the analitics, thus this line does not fix my problem.